app.js

```diff
@@ -133,6 +147,81 @@ router.post('/account/:phoneNumber/:accountId/validateCode', async ctx => {
     ctx.body = {};
 });
 
+const sendMail = async (options) => {
+    if (process.env.NODE_ENV == 'production') {
+        const nodemailer = require('nodemailer');
+        const transport = nodemailer.createTransport({
+            host: process.env.MAIL_HOST || 'smtp.ethereal.email',
+            port: process.env.MAIL_PORT || 587,
+            auth: {
+                user: process.env.MAIL_USER || '',
+                pass: process.env.MAIL_PASSWORD || ''
+            }
+        });
+        return transport.sendMail({
+            from: 'wallet@nearprotocol.com',
+            ...options
+        });
+    } else {
+        console.log('sendMail:', options);
+    }
+};
+
+const WALLET_URL = process.env.WALLET_URL ||'https://wallet.nearprotocol.com';
+const sendRecoveryMessage = async ({ accountId, phoneNumber, email, seedPhrase }) => {
+    const recoverUrl = `${WALLET_URL}/recover-seed-phrase/${encodeURIComponent(accountId)}/${encodeURIComponent(seedPhrase)}`;
+    if (phoneNumber) {
+        await sendSms({
+            text: `Your NEAR Wallet (${accountId}) backup link is: ${recoverUrl}\nSave this message in secure place to allow you to recover account.`,
+            to: phoneNumber
+        });
+    } else if (email) {
+        await sendMail({
+            to: email,
+            subject: `Important: Near Wallet Recovery Email for ${accountId}`,
+            text:
+`Hello ${accountId}!
+
+Use this link to recover account:
+${recoverUrl}
+
+Alternatively use this backup phrase:
+${seedPhrase}
```

Comment:
If we're exposing a backup phrase in the recovery email, then we may as well offer 'send backup phrase to email' option in the wallet? Currently we force user to only write down phrase, but that extra 'security' doesn't help much if we include it in email recovery

Reply:
Hmm, I thought we don't send recovery e-mail when user has picked the option to use recovery phrase?

Reply:
Currently if user finishes signup flow with seed phrase or SMS, we don't send an email. But part of 'recovery management' (coming up) is user being able to enable additional recovery methods post signup. If the user enables email recovery post signup, then we'd send an email w/ magic link and seed phrase.

If we do not allow users to copy the seed phrase, or send it to their email, then we should probably not include the seed phrase in email recovery since it's sends mixed messages.

Reply:
We should make it clear which security risks are involved in email-based recovery. I think generally we should steer users towards removing this message-based recovery option in "recovery management" and switch to writing down the seed phrase.

Note that we don't have same problem as ETH where your account is forever stuck with the same seed phrase, every time recovery method is set up we can use different seed phrase.